timbermill-java/timbermill-client/src/main/java/com/datorama/oss/timbermill/TimbermillBootstraperService.java

```diff
@@ -0,0 +1,90 @@
+package com.datorama.oss.timbermill;
+
+import com.datorama.oss.timbermill.pipe.TimbermillServerOutputPipeBuilder;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+@Service("timbermill")
+public class TimbermillBootstraperService {
+
+	private static final Logger LOG = LoggerFactory.getLogger(TimbermillBootstraperService.class);
+
+	@Value("${timbermill.log.enabled:false}")
```

Comment:
suggestion, you can replace this with a condition on the bean itself 
```suggestion
	@ConditionalOnProperty(name = "timbermill.log.enabled", matchIfMissing = false, havingValue = "true")
```

it would be cleaner but no logs in case it's disabled..

Reply:
good idea but I like the log